src/main/java/Parser.java

```diff
@@ -0,0 +1,89 @@
+/**
+ * Represents a parser that takes in user-typed commands and
+ * returns the equivalent Command object.
+ */
+public class Parser {
+    /**
+     * Returns a Command object based on the interpretation of the user-typed command.
+     *
+     * @param command the user-typed command
+     * @return
```

Comment:
I think you may have forgotten to input the @return text! Other than that its good!